plerkle_messenger/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "plerkle_messenger"
 description = "Metaplex Messenger trait for Geyser plugin producer/consumer patterns."
-version = "1.8.0"
+version = "1.8.1"
```

Just to keep it consistent
```suggestion
version = "1.9.0"
```

---

+1